Source/SlashCmd/Commands.lua

```diff
@@ -40,6 +40,22 @@ function Auctionator.SlashCmd.CleanReset()
   Auctionator.SlashCmd.ResetDatabase()
 end
 
+function Auctionator.SlashCmd.Config(name)
+  if name == nil then
+    Auctionator.Utilities("Current config:")
```

Comment:
Should this be `Auctionator.Utilities.Message`?

Reply:
Yeah, it should be